app/sitemap.ts

```diff
@@ -5,6 +5,7 @@ import {
   DYNAMIC_GENERATED_ROUTES,
   BASE_PATH,
   BASE_URL,
+  EXTERNAL_LINKS_SITMAP,
```

Comment:
```suggestion
  EXTERNAL_LINKS_SITEMAP,
```

Typo on the name of the variable, please be more careful.